### PR TITLE
Add admin controller to standardize scores

### PR DESCRIPTION
**Pull request by thomasmunduchira**

### CODE REVIEW

File: src/models/Application.py [hunk 1]
```diff
@@ -112,3 +130,38 @@ def update_score(user_id, score):
             raise e
 
         return application
+
+    @staticmethod
+    def get_scored_applications():
+        """Returns all scored applications"""
+        return db.session.query(Application) \
+            .filter(Application.score != 0) \
+            .all()
```
Comment (klanmiko) [inline src/models/Application.py]: can we not load everything into memory, can we do this with a mega SQL query, I'm pretty sure we can using a group by

File: src/models/Application.py [hunk 2]
```diff
@@ -112,3 +130,38 @@ def update_score(user_id, score):
...
+        """Updates the standardized scores for all applications"""
+        scored_applications = Application.get_scored_applications()
+
+        # gets all unique user ids
+        user_ids = {application.scoring_user_id for application in scored_applications}
+
+        # creates a dict to map between user_id and list of applications scored by that user
+        user_to_applications = {
+            user_id: list(
+                filter(lambda application: application.scoring_user_id == user_id,
```
Comment (klanmiko) [inline src/models/Application.py]: this can be a db query iteratively

File: src/models/Application.py [hunk 3]
```diff
@@ -112,3 +130,38 @@ def update_score(user_id, score):
...
+                       scored_applications))
+            for user_id in user_ids
+        }
+
+        # creates a dict to map between user_id and list of applications scored by that user with standardized_scores set
+        user_to_applications_with_standardized_scores = {
+            user_id: Application.set_standardized_scores(applications)
+            for user_id, applications in user_to_applications.items()
```
Comment (klanmiko) [inline src/models/Application.py]: this dict isn't used, again, if we do this as a single query and we can do it as one big transaction that would be more elegant

**Comment by klanmiko** [comment]
Also, no scipy

**Comment by thomasmunduchira** [comment]
Created an issue to deal with the optimization later in light of the incoming deadline.

**Pull request merged**